### PR TITLE
Add SPDR space

### PR DESCRIPTION
**Pull request by flisko**
Created a space via ENS.

### CODE REVIEW

File: skins/spiderdao.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.example {
```
Comment (gokaiorg) [inline skins/spiderdao.scss:1]: The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.spiderdao {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
There is a conflict in your PR, can you resolve it?

**Comment by flisko** [comment]
> There is a conflict in your PR, can you resolve it?

Resolved.

**Comment by flisko** [comment]
@gokaiorg Please check.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**